test/TesterInternal/MembershipTests/MembershipTableTestsBase.cs

```diff
@@ -243,52 +268,49 @@ protected virtual string GetAdoInvariant()
                 Assert.NotNull(etagBefore);
 
                 logger.Info("Calling UpdateRow with Entry = {0} correct eTag = {1} old version={2}", siloEntry,
-                    etagBefore, tableVersion);
-
-                ok = await membershipTable.UpdateRow(siloEntry, etagBefore, tableVersion);
-
-                Assert.False(ok, $"row update should have failed - Table Data = {tableData}");
+                    etagBefore, tableVersion != null ? tableVersion.ToString() : "null");
 
+                if (extendedProtocol)
+                {
+                    ok = await membershipTable.UpdateRow(siloEntry, etagBefore, tableVersion);
```

Comment:
Ok, on this test I actually added more things on the IF because on extended protocol, the tableVersion is taken in consideration while performing the update. If I remove the IF, the update will happen since the etag is correct and I ignore the tableVersion by not implement the extended protocol and all the following assertion will fail.